macx/scripts/osxdist.py

```diff
@@ -129,12 +129,12 @@ def copy_resources(self, rsrcs):
 
 	def copy_codecs(self):
 		'''
-			Try to copy the dynamic CELT libraries into the App Bundle.
+			Try to copy the dynamic audio codec libraries into the App Bundle.
 		'''
-		print ' * Attempting to copy CELT libraries into App Bundle'
+		print ' * Attempting to copy audio codec libraries into App Bundle'
 		dst = os.path.join(self.bundle, 'Contents', 'Codecs')
 		os.makedirs(dst)
-		codecs = ('release/libcelt0.0.7.0.dylib', 'release/libcelt0.0.11.0.dylib')
+		codecs = ('release/libcelt0.0.7.0.dylib', 'release/libcelt0.0.11.0.dylib', 'release/libopus.dylib')
```

Comment:
There's a listing of binaries to sign further down in this file. Does libopus.dylib not also need to be listed there?

Reply:
Definitely, I forgot to add it.